### PR TITLE
Condition can be an expression only

### PR DESCRIPTION
**Pull request by mvorisek**
image

### CODE REVIEW

File: src/Selenium2Driver.php [hunk 1]
```diff
@@ -972,7 +972,7 @@ public function evaluateScript($script)
      */
     public function wait($timeout, $condition)
     {
-        $script = "return $condition;";
+        $script = 'return (' . $condition . ')';
```
[the remark this answers is not in the document]
Comment (mvorisek) [reply src/Selenium2Driver.php:975]: @aik099 @stof ok so and can we merge it?

[the remark this answers is not in the document]
Comment (aik099) [reply src/Selenium2Driver.php:975]: Yes, please add `;` and I'll merge the PR.

[the remark this answers is not in the document]
Comment (mvorisek) [reply src/Selenium2Driver.php]: done

**Comment by aik099** [comment]
Merging, thanks @mvorisek .

**Comment by alexpott** [comment]
This change has broken Drupal's test suite and I think is an slight API change - see #324. Inputs that used to work no longer do.

**Pull request merged**